setup.py

```diff
@@ -7,7 +7,8 @@
         'csv': ['unicodecsv'],
         'cli': ['click', 'requests'],
         'html': ['lxml'], # apt: libxslt-dev libxml2-dev
-        'xls': ['xlrd', 'xlwt'], }
+        'xls': ['xlrd', 'xlwt'],
+        '_pandas': ['pandas'], }
```

Comment:
Please change the key to `pandas`, since this is the string user will pass to `pip` to install the dependency (and won't be in conflict with the Python module), like:

```bash
pip install rows[pandas]
```